streams/src/main/java/org/apache/kafka/streams/StreamsMetrics.java

```diff
@@ -26,5 +26,5 @@
 
     Sensor addLatencySensor(String scopeName, String entityName, String operationName, String... tags);
 
-    void recordLatency(Sensor sensor, long startNs, long endNs);
+    void recordLatencyNs(Sensor sensor, long startNs, long endNs);
```

Comment:
Actually for latency I think it makes more sense to use nanos, as it is supposed to be much less than one millisecond, otherwise the maximum throughput will be bounded as 1000 QPS right?